src/app/components/data/shared/rendering-container-image-picker/rendering-container-image-picker.component.ts

```diff
@@ -0,0 +1,151 @@
+import {
+    ChangeDetectionStrategy, ChangeDetectorRef, Component, Input,
+    OnChanges, OnDestroy, forwardRef,
+} from "@angular/core";
+import {
+    ControlValueAccessor,
+    FormControl,
+    NG_VALUE_ACCESSOR,
+} from "@angular/forms";
+
+import { RenderApplication, RenderEngine, RenderingContainerImage } from "app/models/rendering-container-image";
+import { RenderingContainerImageService } from "app/services";
+import { BehaviorSubject, Observable, Subject, combineLatest } from "rxjs";
+import { map, switchMap, takeUntil } from "rxjs/operators";
+import "./rendering-container-image-picker.scss";
+
+@Component({
+    selector: "bl-rendering-container-image-picker",
+    templateUrl: "rendering-container-image-picker.html",
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    providers: [
+        {
+            provide: NG_VALUE_ACCESSOR, useExisting: forwardRef(() =>
+                RenderingContainerImagePickerComponent), multi: true,
+        },
+        RenderingContainerImageService,
+    ],
+})
+
+export class RenderingContainerImagePickerComponent implements ControlValueAccessor, OnChanges, OnDestroy {
+
+    public get appDisplay() {
+        return this.UpperCaseFirstChar(this.app);
+    }
+
+    public get renderEngineDisplay() {
+        return this.UpperCaseFirstChar(this.renderEngine);
+    }
+
+    public appVersionControl = new FormControl();
```

Comment:
is there any validation to do? like required?

Reply:
Handled by default - I believe because containerImage is a template parameter and requires a value before template validation will succeed and "Create" buttons are enabled